packages/dialect-react-sdk/hooks/useSingleFeed.ts

```diff
@@ -0,0 +1,51 @@
+import type { DappMessage, FindDappMessageQuery } from '@dialectlabs/sdk';
+import { useMemo } from 'react';
+import useSWR from 'swr';
+import { EMPTY_ARR } from '../utils';
+import { SINGLE_FEED_CACHE_KEY_FN } from './internal/swrCache';
+import useDialectSdk from './useDialectSdk';
+
+interface UseSingleFeedParams {
+  nonVerifiedDapps?: boolean;
+  take?: number;
+  skip?: number;
+  pollingInterval?: number;
+}
+
+interface UseSingleFeedValue {
+  data: DappMessage[];
+  isLoading: boolean;
+}
+
+const useSingleFeed = ({
+  nonVerifiedDapps = false,
+  take = 25,
+  skip = 0,
+  pollingInterval,
+}: UseSingleFeedParams = {}): UseSingleFeedValue => {
+  const sdk = useDialectSdk();
+
+  const query: FindDappMessageQuery = useMemo(
```

Comment:
I think this useMemo is unnecessary, since either way it's going to the SINGLE_FEED_CACHE_KEY_FN which converts it to a string and memoizes the response.

I believe you can just do this on line 38:
```
SINGLE_FEED_CACHE_KEY_FN({ dappVerified: !nonVerifiedDapps, take, skip })
```

Reply:
this is good catch, memo removed